app/src/components/MapIFrame.jsx

```diff
@@ -3,8 +3,17 @@ import Modal from './Shared/Modal';
 import NoLogin from '../containers/Map/NoLogin';
 import Header from '../containers/Header';
 import FooterMini from './Shared/FooterMini';
+import mapStyles from '../../styles/components/c-map.scss';
 
 class MapIFrame extends Component {
+
+  constructor(props) {
```

Comment:
this does not need to be state, it doesn't change during the component lifecycle. you can use just props.